WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -57,7 +57,7 @@
       <span class="caption">
         <% if @past_selected %>
           <% #Here params[:year] is a string, so we need regexp %>
```

Comment:
Ah, that is true. I guess I was getting hung up on the mention of a regexp that is no longer here.